ndcube/utils/wcs.py

```diff
@@ -106,6 +106,41 @@ def _augment(cls, header, naxis):
             newheader['CTYPE' + axis] = projection
         return newheader
 
+    def dependent_axes(self, axis):
```

Comment:
I think this should be a function not a method of the WCS class, just incase someone is using `astropy.wcs.WCS`.